pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -160,6 +160,8 @@ type TensorflowSpec struct {
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
 	// Defaults to requests and limits of 1CPU, 2Gb MEM.
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// Allow user to change between REST or gRPC protocol.
+	Protocol string `json:"protocol,omitempty"`
```

Comment:
I think we can define a type `type Protocol string` here and add marker `+kubebuilder:validation:Enum=http;gRPC` to have kubebuilder generate the openAPI scheme validation. The default value is http.

https://book.kubebuilder.io/reference/generating-crd.html#validation

Reply:
done

Reply:
I named it rest;gprc to make the port reference consistent.